lib/shoryuken/polling/strict_priority.rb

```diff
@@ -38,6 +38,11 @@ def active_queues
           .reverse
       end
 
+      def unpause_queue(queue)
```

Comment:
For symmetry with the method `messages_found`, let's consider renaming this method `message_processed`.

The reason is that the interaction between the `Manager` class and the polling strategy is that the manager is just informing the polling strategy of certain events, allowing the polling strategy to respond as it sees fit. In this case, it's not the responsibility of the `Manager` to tell the polling strategy to _pause_ a queue, it's just the responsibility for the `Manager` to let the polling strategy know that it's done processing the message: the polling strategy is welcome to do whatever it wants with that knowledge.

Reply:
Let's also add `message_processed` to `Shoryuken::Polling::BaseStrategy`. Its implementation can be empty (for backward compatibility--users might have subclassed the base class themselves).